### PR TITLE
Updating DevPurchaceClient with Valid Transaction/Receipt

### PR DESCRIPTION
**Pull request by mattsac**

### CODE REVIEW

File: src/purchase-client.ts [hunk 1]
```diff
@@ -177,15 +178,47 @@ export class DevPurchaseClient implements PurchaseClient {
...
+      const testDate = new Date();
+
+      const jwtHeader = {
+        "alg": "HS256",
+        "typ": "JWT"
+      };
+
+      const jwtPayload = {
+        "topic": "bits_transaction_receipt",
+        "exp": Math.round(testDate.getMilliseconds() + 10000 / 1000),
```
Comment (camr) [inline src/purchase-client.ts:190]: Why the `10000 / 1000`? Did you want `(testDate.getMilliseconds() + 10000) / 1000`?

Also, I believe you want `testDate.valueOf()` here. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMilliseconds

File: src/purchase-client.ts [hunk 2]
```diff
@@ -177,15 +178,47 @@ export class DevPurchaseClient implements PurchaseClient {
...
+        "data": {
+          "transactionId": `test:${testDate}`,
```
Comment (camr) [inline src/purchase-client.ts:192]: Should you use `testDate.valueOf()` here?

File: src/purchase-client.ts [hunk 3]
```diff
@@ -177,15 +178,47 @@ export class DevPurchaseClient implements PurchaseClient {
...
+      const encodedPayload = btoa(JSON.stringify(jwtPayload));
+
+      // End of signature intentionally doesn't match.
+      const testEncodedJWT = `${encodedHeader}.${encodedPayload}.DK02m0j0HQMKsPeFIrAuVdFh5X8f0hknjEKHAjGt6B0`;
+
       setTimeout(() => {
         if (item) {
           const tx: Transaction = {
-            transactionId: 'dev-transaction-id',
+            transactionId: `test:${testDate}`,
```
Comment (camr) [inline src/purchase-client.ts:216]: Same question about `testDate.valueOf()`.

File: src/purchase-client.ts (hunk 1 above)
Comment (mattsac) [reply src/purchase-client.ts:190]: Ah.  yes on both.

**Review by camr — APPROVED** [review]

**Pull request merged**